---
int32[] order_ids
string[] items_list